client/question/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -53,6 +55,28 @@ tie.factory('PythonPrereqCheckService', [
       });
     };
 
+    /**
+     * Processes studentCode and converts all escaped characters, then all
+     * single quote + double quote str to filler character "x".
+     * This is to preserve character length for error checks.
+     *
+     * @param {string} [code] The code to convert
+     * @return {string} code where all 'foo' & "bar" converted to xxxxx & xxxxx
+     */
+    var getObscuredCode = function(code) {
+      var escapeRegexp = new RegExp(/\\[\w,',",\\]/, 'g');
```

Comment:
Why do you need the commas in this regex?

Reply:
I guess I wasn't aware the commas were optional. Probably in part due to the tools and documentation suggesting them by default. Works without em? Done.